### ISSUE TITLE
Cut a 0.4.8 release

### ISSUE THREAD

**Issue by abadger** [wearpants/twiggy#60]
I've discovered a "release bug"....  twiggy 0.4.7 is the version listed on pypi.  However, comparing the contents of the tarball with the git repository I found that the code there is from a checkout of master ( e64866e92bcd69f20e16e11c3e23a595afb08d46 ), not the maint-0.4 branch.  Since it's been 2 years since the 0.4.7 release went up on pypi, maybe you just want to make 0.5.0 official? Bugs or not, finished or not, people have either been using it with the new API all this time or not using it at all.

Regardless of your decision on that... if you would at least be willing to get the docs and pypi in sync that would be great.  I scratched my head for a while this weekend trying to figure out why http://twiggy.readthedocs.io/en/latest/  did not match up with the code that I had installed when both claimed to be for 0.4.7.

**Comment by wearpants** [comment]
Thanks for all this, it's very exciting to have someone express an interest
- lack of feedback was part of the reason I moved away from this project -
didn't know anyone nwas using it until several years later! (also the Arab
Spring happened, but that's another story).

I recently stumbled on twiggy goodies and been thinking about picking this
up again, including much of what you've just PR'd - a friend to help out
would definitely be motivating!

More later today, it's early here. :-)
